recipes/giotto-tda/meta.yaml

```diff
@@ -0,0 +1,76 @@
+{% set name = "giotto-tda" %}
+{% set version = "0.2.2" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: "https://github.com/giotto-ai/{{ name }}/archive/v{{ version }}.tar.gz"
+  sha256: 36e637a4cc9beb9454932534dbc790f0871698230546737e3eba101dd3ac1b4e
+
+build:
+  number: 0
+  skip: true  # [py2k]
+  skip: true  # [win and vs<17]
+  script:
+    - "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+  host:
+    - python
+    - pybind11
+    - boost >=1.56.*
+    - setuptools
+    - pip
+    - numpy >=1.17.0
+    - scipy >=0.17.0
+    - joblib >=0.13
+    - scikit-learn >=0.22.0
+    - python-igraph >=0.7.1.post6
```

Comment:
Please remove the post part of these deps

Reply:
@scopatz FYI, this dependency has now been bumped to `0.8.2`.